src/renderers/WebGLRenderer.js

```diff
@@ -1144,6 +1144,20 @@ function WebGLRenderer( parameters ) {
 
 		}
 
+		if ( _currentRenderTarget !== null && _currentRenderTarget.isWebGLRenderTargetCube ) {
+
+			var textureProperties = properties.get( _currentRenderTarget.texture );
+			_gl.framebufferTexture2D(
```

Comment:
I guess you need to cache the current active cube face, otherwise  `framebufferTexture2D()` is called at each invocation of `WebGLRenderer.render()` even if the activeCubeFace has not changed.